package.json

```diff
@@ -105,6 +107,8 @@
     "sinon": "^7.5.0",
     "sinon-chai": "^3.4.0",
     "tar": "^6.0.1",
+    "tar-fs": "^2.1.0",
+    "tmp-promise": "^3.0.2",
```

Comment:
This adds a few things here, but the code below doesn’t reference them … am I missing something?

Reply:
good point 😅 

this PR used to fetch a new version of curator because our build variants didn't have it. But the build folks updated the variants, so we no longer have to! I'll remove these.